### ISSUE TITLE
Update animated gif with 5 presses

### ISSUE THREAD

**Issue by jmatsushita** [PanicInitiative/PanicButton#107]
The current one only instructs to do 4 presses before confirming.

**Comment by niccs** [comment]
@jmatsushita ,I have observed one point:--

There are two files in assets/media/mobile folder-
1) panic_button_activation_360.gif --- used only for English language and
2)panic_button_activation.png  --used for all other languages

So ideally, the gif file is not played if the setup is done from any other language (may be designed so  because the text strings used in gif are in english). You can verify this behavior of the app, from the latest playstore version.
Please comment.

**Comment by jmatsushita** [comment]
Good point we should localise the button. Could you please for now make sure there is a different gif file name for each language (just copying the latest version)? I will then change the file when it is localised. Thanks!

**Comment by jimmylogan** [comment]
Presume that further changes will need to be made after #178 is implemented.

**Comment by jmatsushita** [comment]
Committed new 7 presses default in  1e7521b

**Comment by jmatsushita** [comment]
@niccs There is probably a bit of new logic to create in order to display the new animations. Can we create new JSON objects for each trigger option (7 press, 5 press + confirmation, custom) and dynamically link to the one that needs to be displayed so we can still configure the image and text in the JSON?

**Comment by niccs** [comment]
@jmatsushita , I think we can dynamically pick the images. json will just parse and store the name of image in sqliteDB, and while fetching we can change that. No need of creating new json objects. We are just using this in hardware training screen.

**Comment by niccs** [comment]
@jmatsushita ,

As per our discussion, I am putting down three cases:

Case -1) default 7 repeated press:-- The app will pick the locale specific .gif file name from mobile_xx.json file and play it. The file name is:--  panic_button_activation_360_xx.gif

Case -2) 5 Presses plus confirmation:-  We need the resource name. Right now, I am using-- panic_button_activation_5press_confirm_xx.gif . and will checkin the same default existing gif fille change its name.

Case-3) Custom settings:-- It will take the static image panic_button_activation.png and with that a toast message  is displayed 
a) --" X Repeated presses and then press for confirmation within 3 seconds"--(in case confirmation is activated)
b)  --" X Repeated presses "--(in case confirmation is not activated))
